terraform/stacks/data_archive/raw_data_archive.tf

```diff
@@ -0,0 +1,75 @@
+################################################################################
+# Buckets
+
+resource "aws_s3_bucket" "raw_data_bucket" {
+  bucket = var.raw_data_bucket_name
```

Comment:
This `var.raw_data_bucket_name` is not found yet.

Perhaps, there is `variables.tf` update coming in next commit?

Reply:
Huh?
Not sure what happened there... the variables file was already there, I just added an entry for the oncoanalyser bucket.

Reply:
Argh, changing IDEs... VSCode does not autosave changes...